rust/core-lib/Cargo.toml

```diff
@@ -28,8 +29,7 @@ version = "3.0"
 default-features = false
 features = ["assets","dump-load-rs","dump-create"]
 
-[target."cfg(target_os = \"fuchsia\")".dependencies]
-sha2 = "0.7"
+#[target."cfg(target_os = \"fuchsia\")".dependencies]
```

Comment:
oops?

Reply:
We hard depend on sha2 0.8 anyway, so we don't need to depend on 0.7 here. Crosscompiled to x86_64-fuchsia to test the change and it compiled just fine.

Reply:
Ah didn't notice this was related to sha2. We can just delete this line I think.

Reply:
Ah, but there are some more commented lines about fuchsia below that, didn't want to destroy the context for thos.